pkg/app/pipedv1/plugin/kubernetes/deployment/server.go

```diff
@@ -189,7 +189,7 @@ func (a *DeploymentService) loadManifests(ctx context.Context, deploy *model.Dep
 	manifests, err := a.loader.LoadManifests(ctx, provider.LoaderInput{
 		PipedID:          deploy.GetPipedId(),
 		AppID:            deploy.GetApplicationId(),
-		CommitHash:       deploy.GetTrigger().GetCommit().GetHash(),
+		CommitHash:       deploymentSource.GetRevision(),
```

Comment:
This could be the same but I like the previous version more. How about to rename `GetRevision` to `GetCommitHash` 🤔

Reply:
First, I also like `commit hash` more than `revision`.

However, this is not the same because the `deploymentSource` can be the running deploy source, not the target deploy source.

The naming `GetRevision` comes from proto definition, so how about renaming this field as `commit_hash`?
https://github.com/pipe-cd/pipecd/blob/17a5007b6f1ab40dd230376a511b2f1e1d109b40/pkg/plugin/api/v1alpha1/deployment/api.proto#L142-L143

Reply:
I got your point. Also, it's a commit hash, so it should be named directly as a commit hash to reduce confusion. Let's address it by another PR (change proto and update this)